app/controllers/submit_form_controller.rb

```diff
@@ -6,6 +6,7 @@ class SubmitFormController < ApplicationController
   around_action :with_browser_locale, only: %i[show completed success]
   skip_before_action :authenticate_user!
   skip_authorization_check
+  skip_before_action :verify_authenticity_token, only: :update
```

Comment:
Is this just temporary, or do we eventually want to rethink removing CSRF for updates?